### PR TITLE
🐛 Fix chai assertion wrapper bug

### PR DESCRIPTION
**Pull request by estherkim**
Fixes this error seen when running `gulp e2e`
```
(node:141083) UnhandledPromiseRejectionWarning: AssertionError: expected { Object (bottom, height, ...) } to have property 'x' of 0, but got 800
(node:141083) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
(node:141083) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Also removes some duplicates.

### CODE REVIEW

File: build-system/tasks/e2e/expect.js [hunk 1]
```diff
@@ -109,15 +109,11 @@ const chaiMethodsAndProperties = [
...
   {name: 'length', type: ChaiType.CHAINABLE_METHOD},
-  {name: 'length', type: ChaiType.CHAINABLE_METHOD},
-  {name: 'lengthOf', type: ChaiType.CHAINABLE_METHOD},
   {name: 'lengthOf', type: ChaiType.CHAINABLE_METHOD},
   {name: 'lessThan', type: ChaiType.METHOD},
   {name: 'lt', type: ChaiType.METHOD},
   {name: 'lte', type: ChaiType.METHOD},
   {name: 'match', type: ChaiType.METHOD},
-  {name: 'match', type: ChaiType.METHOD},
-  {name: 'matches', type: ChaiType.METHOD},
```
Comment (cvializ) [inline build-system/tasks/e2e/expect.js]: 😬

File: build-system/tasks/e2e/expect.js [hunk 2]
```diff
@@ -127,7 +123,7 @@ const chaiMethodsAndProperties = [
   {name: 'oneOf', type: ChaiType.METHOD},
   {name: 'ownProperty', type: ChaiType.METHOD},
   {name: 'ownPropertyDescriptor', type: ChaiType.METHOD},
-  {name: 'property', type: ChaiType.METHOD},
+  {name: 'property', type: ChaiType.CHAINABLE_METHOD},
```
Comment (cvializ) [inline build-system/tasks/e2e/expect.js]: Why should this be chainable method? It's not implemented that way in Chai

https://github.com/chaijs/chai/blob/42509fac016182ddde8b4edf9ff3ff1ce285cebf/lib/chai/core/assertions.js#L1885

```js
Assertion.addMethod('property', assertProperty);
```

Comment (estherkim) [reply build-system/tasks/e2e/expect.js]: I'm not sure, actually. I narrowed down the cause of the error to this line and found that changing the method fixed it.

Comment (cvializ) [reply build-system/tasks/e2e/expect.js]: huh weird. I think a comment could help if there's weirdness with this later, but if you're ready to merge now don't let that stop you

Comment (estherkim) [reply build-system/tasks/e2e/expect.js]: Good idea, added a comment! Thanks.

**Pull request merged**